examples/inspection/plot_permutation_importance.py

```diff
@@ -130,8 +130,7 @@
 y_ticks = np.arange(0, len(feature_names))
 fig, ax = plt.subplots()
 ax.barh(y_ticks, tree_feature_importances[sorted_idx])
-ax.set_yticklabels(feature_names[sorted_idx])
-ax.set_yticks(y_ticks)
+
```

Comment:
You can get the tick labels for the y-axis back, without warning, by
```python
from matplotlib.ticker import FixedLocator
``` 
and then
```suggestion
ax.yaxis.set_major_locator(FixedLocator(y_ticks))
ax.set_yticklabels(feature_names[sorted_idx])
```

Reply:
An alternative would be to change the order:

```py
ax.set_yticks(y_ticks)
ax.set_yticklabels(feature_names[sorted_idx])
```

Reply:
Thanks @lorentzenchr I will do that now

Reply:
Thanks @thomasjpfan I really appreciate the help